### ISSUE TITLE
Howling Fjord Part 1 - Various issues, quests and mobs

### ISSUE THREAD

**Issue by jack40melb** [gamefreedomgit/Frostmourne#427]
[Fjord Hawk](https://www.wowhead.com/wotlk/npc=24747/fjord-hawk) resting in mid air
![Screenshot 2024-05-11 104610](https://github.com/gamefreedomgit/Frostmourne/assets/6733191/9d096a0f-ab49-47f6-b3dd-8d40846c9637)

[Quest : Iron rune constructs and you collecting data](https://www.wowhead.com/wotlk/quest=11489/iron-rune-constructs-and-you-collecting-data) Visual bug. Missing blue crystal with 2 spinning globes near wagon. **Quest still completes.**
[You can see it in this video](https://www.youtube.com/watch?v=mWDmxo1Du3Q)
![image_2024-05-12_001744953](https://github.com/gamefreedomgit/Frostmourne/assets/6733191/7beae1b6-b4be-477a-a825-e00d59925b3b)

[Quest; iron-rune-constructs-and-you-the-bluff](https://www.wowhead.com/wotlk/quest=11491/iron-rune-constructs-and-you-the-bluff) Missing chat dialogue from Iron Rune Construct as it nears the carpet. You only see the Lebronski's chat
[See video](https://youtu.be/US3ENt9loBs?t=33)
![Screenshot 2024-05-11 181305](https://github.com/gamefreedomgit/Frostmourne/assets/6733191/837d3c4a-31d1-4df4-b07a-bb64f76213b3)

[Quest: the-delicate-sound-of-thunder](https://www.wowhead.com/wotlk/quest=11495/the-delicate-sound-of-thunder)  
- **Can not be completed**. Statue/npc thingy doesn't appear when you reach location to complete quest 
- the group of dwarfs that appear from top path fall down through the ground at location where they are supposed to kneel
- When you enter the tunnel, the construct agros all mobs. It has no attack except the Bluff ability (see below)
- Construct's **Rocket jump** doesnt seem to work. you get a speed boost but nothing else to throw you to next location.
- Constructs Bluff doesnt seem to affect dwarfs inside tunnel. as well as some below
- [how it should be](https://youtu.be/_wmU9qV8CWA?t=37)
- ![Screenshot 2024-05-12 000058](https://github.com/gamefreedomgit/Frostmourne/assets/6733191/8b01c7a5-2c6b-4325-aee1-3dfb5070f206)

[Quest: the-staff-of-storms-fury](https://www.wowhead.com/wotlk/quest=11511/the-staff-of-storms-fury)
- Missing lightning effect on all decks of ship
- Missing dead bodies on all decks of ship
- missing npc [Abdul the Insane](https://www.wowhead.com/wotlk/npc=24900/abdul-the-insane) on patrolling the top deck
- Missing ligtning going  off from The Staff of storms on lower deck
- [See video ](https://youtu.be/0EE-nDAZipo?t=37)
![Screenshot 2024-05-11 225141](https://github.com/gamefreedomgit/Frostmourne/assets/6733191/1b5e4de5-34ea-4404-899d-da4d2c699a97)

 

[Quest: jack-likes-his-drink](https://www.wowhead.com/wotlk/quest=11466/jack-likes-his-drink)

- NPC [Jack Adams](https://www.wowhead.com/wotlk/npc=24788/jack-adams) Should be dancing on top of table
- Missing dialogue from Jack adams and druken sailors in room
- [See Video](https://youtu.be/ckdNwQd2pKU?t=55)

**Very low Drop rate** 
- [bear-musk](https://www.wowhead.com/wotlk/item=34084/bear-musk) for the [Quest: the-fragrance-of-money](https://www.wowhead.com/wotlk/quest=11455/the-fragrance-of-money)
-[ island-shoveltusk-mea](https://www.wowhead.com/wotlk/item=36776/island-shoveltusk-meat)t for the [Quest: feeding-the-survivors](https://www.wowhead.com/wotlk/quest=11456/feeding-the-survivors). Cleared both islands of all mobs and only got 2 of each.

[Quest: dead-mans-debt](https://www.wowhead.com/wotlk/quest=11467/dead-mans-debt) short a few more [Forlorn Souls](https://www.wowhead.com/wotlk/npc=24789/forlorn-soul) at location

**Comment by Raulfin** [comment]
I can't even get the chat dialog for [Quest; iron-rune-constructs-and-you-the-bluff](https://www.wowhead.com/wotlk/quest=11491/iron-rune-constructs-and-you-the-bluff) so the zone is basically soft locked there for me.

**Comment by DiyaniWoW** [comment]
For the quest The Delicate Sound of Thunder, the construct's boosters didn't work; got a message "Requires Iron Dwarf Transport Rune". Not sure if this is what bugged the whole quest for me, but when I got to the cave all the mobs would phase in/out. I ran to the spot for completion but instead of completion was teleported back to the closest elevator. One of those times it looked like I was falling through the map. After a number of attempts I logged out, only to log back in falling to my death. Found my corpse at the top of the elevator and tried again 3 times before getting permanently stuck at the bottom of the elevator after getting teleported. I was still in the construct, but could see my normal skills with the construct skills but couldn't exit the construct. While chatting with the GM, I somehow managed to exist the construct only to find my character invisible and still stuck on the elevator. Took a relog for whatever the GM did to get me unstuck.
